content/docs/1.7.0/snapshots-and-backups/csi-volume-clone.md

```diff
@@ -41,6 +43,38 @@ spec:
 > In addition to the requirements listed at [CSI volume cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/),
 > the `cloned-pvc` must have the same `resources.requests.storage` as the `source-pvc`.
 
+
+## Clone a Volume Using the Longhorn UI
```

Comment:
Would you like to mention buck cloning?

Reply:
Sure, let me add another block for bulk volume cloning.